target/scripts/build/packages.sh

```diff
@@ -37,6 +37,9 @@ function _install_postfix
   chmod +x /bin/hostname
   apt-get "${QUIET}" install --no-install-recommends postfix
   mv /bin/hostname.bak /bin/hostname
+
+  # We are not using Debian's default `chroot` jail config for Postfix, this config is not needed:
```

Comment:
Rewording to trigger CI:

```suggestion
  # Irrelevant - Debian's default `chroot` jail config for Postfix needed a separate syslog socket:
```